src/stylesheets/components/_inventory.scss

```diff
@@ -0,0 +1,21 @@
+.inventory--MAIN {
+  background-color: $grey-1;
+  overflow: auto;
+
+  .inputfield--MAIN {
+    margin: 1.5rem 0 0 1.5rem;
```

Comment:
rem's are new to me too -- but I think I understand what they mean -- instead of explicit size like PX, then are relative of their parent.  

Is this for responsive app design?